locationConfig.json

```diff
@@ -16,7 +16,7 @@
         "details": {
             "connector": {
                 "sproxyd": {
-                    "bootstrap": ["localhost:8182"]
+                    "bootstrap": ["localhost:8182:/proxy/arc/"]
```

Comment:
As discussed, path should not be set per bootstrap, but per sproxyd. New commits reflect changes